packages/suite/src/services/suite/metadata/DropboxProvider.ts

```diff
@@ -200,6 +200,11 @@ class DropboxProvider extends AbstractMetadataProvider {
             // this should never happen
             message = 'unknown error';
         }
+
+        if (message.includes('Failed to fetch')) {
```

Comment:
Is there a more stable method of detection rather than checking the message?

Reply:
I've tried figuring out but seems like only that message is being returned? And do we have/need a global handler for connection outage? Perhaps if we centralize the state of being w/o a connection it would be easier to handle issues related to it?

Reply:
Would love to see a better solution than this. Some centralized http/fetch module would be nice.

Reply:
Let's discuss it on the next code improvements meeting c: @Nodonisko It can at lest be Axios with some handling logic in a separate config file.